Files/MultilingualResources/Files.hu-HU.xlf

```diff
@@ -1944,21 +1944,13 @@
         </trans-unit>
         <trans-unit id="SettingsUseFileListCache.Header" translate="yes" xml:space="preserve">
           <source>Cache files and folders for better performance</source>
-          <target state="new">Cache files and folders for better performance</target>
+          <target state="translated">Fájlok és mappák gyorsítótárba való helyezése a jobb teljesítményért</target>
         </trans-unit>
         <trans-unit id="ItemAlreadyExistsDialogCloseButtonText" translate="yes" xml:space="preserve">
           <source>Cancel</source>
           <target state="needs-review-translation" state-qualifier="tm-suggestion">Mégse</target>
         </trans-unit>
-        <trans-unit id="StatusCenter.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Ongoing Tasks</source>
-          <target state="new">Ongoing Tasks</target>
-        </trans-unit>
-        <trans-unit id="StatusCenterFlyoutTitle.Text" translate="yes" xml:space="preserve">
-          <source>Ongoing Tasks</source>
-          <target state="new">Ongoing Tasks</target>
-        </trans-unit>
```

Comment:
@KristofKekesi It looks like these strings were accidently removed in your commit, can you revert this change?

Reply:
Sorry for that. I didn't compare before pull.

Reply:
Fixed